src/mlpack/tests/load_save_test.cpp

```diff
@@ -906,6 +905,30 @@ BOOST_AUTO_TEST_CASE(SaveHDF5Test)
 
 #endif
 
+/**
+ *
+ * Test one hot encoding.
+ */
+BOOST_AUTO_TEST_CASE(OneHotEncodingTest)
+{
+  arma::Mat<size_t> matrix;
+  matrix = "1 0;"
+           "0 1;"
+           "1 0;"
+           "1 0;"
+           "1 0;"
+           "1 0;"
+           "0 1;"
+           "1 0;";
+// output matrix
```

Comment:
This is picky but can you make this comment a complete sentence? :)  I am not sure what the exact meaning is.

Reply:
:)